### PR TITLE
More up to date data, quoted entries in dictionaries

### PR DESCRIPTION
**Pull request by zarifmahmud**
Contains data from past few days, and quoted the entries in the dictionary.

### CODE REVIEW

File: data/covid19_model_predictions/covid19_model_predictions_dictionary.csv [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+"lo_0_95_cases","numeric","the lower 95% simultaneous credible interval for daily cases"
+"hi_0_95_cases","numeric","the higher 95% simultaneous credible interval for daily cases"
+"lo_0_25_intensity","factor","intensity associated with the lower 25% simultaneous credible interval for daily cases"
+"hi_0_25_intensity","factor","intensity associated with the higher 25% simultaneous credible interval for daily cases"
+"lo_0_5_intensity","factor","intensity associated with the lower 50% simultaneous credible interval for daily cases"
+"hi_0_5_intensity","factor","intensity associated with the higher 50% simultaneous credible interval for daily cases"
+"lo_0_8_intensity","factor","intensity associated with the lower 80% simultaneous credible interval for daily cases "
+"hi_0_8_intensity","factor","intensity associated with the higher 80% simultaneous credible interval for daily cases"
+"lo_0_95_intensity","factor","intensity associated with the lower 95% simultaneous credible interval for daily cases"
+"hi_0_95_intensity","factor","intensity associated with the higher 95% simultaneous credible interval for daily cases""
```
Comment (rrwen) [inline data/covid19_model_predictions/covid19_model_predictions_dictionary.csv]: @zarifmahmud This line has an extra double quote, please double check your csv files or use a csv linter/online service ([csvlint.io](https://csvlint.io/))

```suggestion
"hi_0_95_intensity","factor","intensity associated with the higher 95% simultaneous credible interval for daily cases"

```

**Review by rrwen — CHANGES_REQUESTED** [review]
@zarifmahmud Looks good so far :+1: - but I found a slight csv format issue with the dictionary csv file, see:

🔧 https://github.com/cghr-toronto/covid-data/pull/11/files#r447256049

It would be good to double check the formatting of your csvs (maybe a unit test in your code), before pushing changes in

❗ Once we establish a set of working updates - lets look into allowing these to be pushed directly into master

File: data/covid19_model_predictions/covid19_model_predictions_dictionary.csv (hunk 1 above)
Comment (zarifmahmud) [reply data/covid19_model_predictions/covid19_model_predictions_dictionary.csv]: @rrwen Sorry about that, I'd initially started manually inserting quotes on this file before I found some vim commands to do it automatically for this one and the rest; this must have been left behind.

Comment (rrwen) [reply data/covid19_model_predictions/covid19_model_predictions_dictionary.csv]: @zarifmahmud No worries - thanks for the fix! 👍

**Review by rrwen — APPROVED** [review]
@zarifmahmud Thanks! Approved and merged to master 👍

**Pull request merged**